src/main/java/seedu/address/model/customer/Customer.java

```diff
@@ -24,6 +27,8 @@ public class Customer {
     // Data fields
     private final Set<Tag> tags = new HashSet<>();
 
+    private final UniqueCommissionList commissions = new UniqueCommissionList();
```

Comment:
I believe this `UniqueCommissionList` could be replaced by a Set since it just represents the data of the commissions and doesn't need to be observable. I also don't think it is a good idea to use this mutable list since changing the commissions in the unique commission list might not change the hash of the Customer and lead to the issue mentioned above in `DeleteCommissionCommand.java`.